pkg/cmd/root/help_topic.go

```diff
@@ -262,30 +276,39 @@ var HelpTopics = map[string]map[string]string{
 	},
 }
 
-func NewHelpTopic(ios *iostreams.IOStreams, topic string) *cobra.Command {
+func NewCmdHelpTopic(ios *iostreams.IOStreams, ht helpTopic) *cobra.Command {
```

Comment:
Injecting a `helpTopic` here is nice because it doesn't rely on some global state (the `HelpTopics`). It's not possible to provide a `topic` that we wouldn't expect to support.